modules/bvl_feedback/test/TestPlan.md

```diff
@@ -1,10 +1,17 @@
 # Behavioural Feedback Test Plan
 
-1. Behavioural feedback button (notepad in the toolbar) should show up on the following pages:
- * Candidate Profile
- * Instrument List
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
+ ![image](images/candidate_list.PNG)  
+ * Instrument List  
+ To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.  
```

Comment:
```suggestion
Feedback on a whole Visit is accessed via the Instrument List page - click the pencil icon in the blue menu bar.
```
Important - for visit-level feedback, use the pencil icon while you're looking at the instrument list module (don't click through to a visit label)